spec.html

```diff
@@ -28857,10 +28874,16 @@ <h1>
         </dl>
         <emu-alg>
           1. Assert: If _direct_ is *false*, then _strictCaller_ is also *false*.
-          1. If _x_ is not a String, return _x_.
+          1. If _x_ is an Object and IsCodeLike(_x_) is *true*, then
+            1. Let _xStr_ be ? ToString(_x_).
```

Comment:
Under what circumstances would this ToString throw? Should it instead be infallible?
```suggestion
          1. If _x_ is an Object and IsCodeLike(_x_) is *true*, then
            1. Let _xStr_ be ! ToString(_x_).
```

Reply:
My understanding is that if an objects to string function throws this operation will too. It also appears to be `? ToString` in the Create dynamic function algorithm too.